Pocket/pocket-ios
# fix(tags): update rename tag functionality

## Summary
Fix rename tag function so that it does not deactivate upon tag selection after renaming 1 tag.

## References 
IN-940

## Implementation Details
* Separated `EditBottomBar` and moved logic into `Textile`. Update alert UI to utilize iOS 16 Alert View with Textfield and remove UIKit implementation. Updated rename tests. Adding `selectedItems.removeAll()` fixes bug where rename breaks after 1 tag is renamed. 

* As a nice polish, also add scrolling ability after a user renames a tag since the list is sorted alphabetically.

* Added analytics for `renameTag` and `deleteTag`

## Test Steps
1. Nav to Archive > Tagged > Edit
2. Select a tag name
3. Rename it
==> after 1 tag is renamed, the 'Rename' function should not break and works as expected upon another tag name selection 

## PR Checklist:

## Screenshots
